centinel/client.py

```diff
@@ -37,7 +37,7 @@ def get_input_file(self, experiment_name):
         return os.path.join(self.config['dirs']['data_dir'], input_file)
 
     def load_input_file(self, name):
-        input_file = self.get_input_file(name)
+        input_file = self.get_input_file("%s.txt" %(name))
```

Comment:
I made this change because the current code is not working at all. If we just left it like this, the baseline experiment is not functional. I didn't know that there could be more extension types, or even no extension.

Changing code to support arbitrary extensions, or just rename world.txt to world on server side are all doable and easy to do. But we should fix this ASAP instead of doing nothing (current client is NOT functional).

By the way, making client support arbitrary extensions seems inferior to me, since
1. this increase complexity of client
2. using a single extension or no extension has no obvious negative effects or overhead. Client could also have a determined file name.

So, in my opinion this is definitely an issue that needs to be addressed. Supporting random extensions, changing data file back to no extension, or using a fixed extension?